### DISCUSSION TITLE
Failed post request for login in example but shows email?

### DISCUSSION THREAD

**Discussion by Dnguye92** [gladly-team/next-firebase-auth#205] [Q&A]
**Describe the bug**
While using the example listed, I get a failed post request when trying to login but I still see my email after I attempted the login?
![image](https://user-images.githubusercontent.com/17717354/121490531-f864f300-c989-11eb-92b9-cb90ffe6f2b9.png)

**Version**
What version of `next-firebase-auth` are you using?
0.13.1

**To Reproduce**
1. Click on "Sign in"
2. Sign in
3. Redirects to `/`.

**Expected behavior**
Not expecting a failed post request.

**Comment by kmjennison** [comment]
This is probably a configuration error for the Firebase settings, as the live [demo app](https://nfa-example.vercel.app/) works. I recommend modifying your demo app API endpoint to log the actual error, then double-checking your Firebase email, private key, etc.
